_episodes/01-variables.md

```diff
@@ -0,0 +1,356 @@
+
+* integer numbers,
+* floating-point numbers, and
+* character strings.
+
+We can use Python as a calculator:
+
+~~~
+3 + 5 * 4
+~~~
+{: .language-python}
+~~~
+23
+~~~
+{: .output}
+
+~~~
+4.5 / 1.5
+~~~
+{: .language-python}
+~~~
+3.0
+~~~
+{: .output}
+
+To create a string, we add single or double quotes around some text:
+
+~~~
+"hello"
+~~~
+{: .language-python}
+~~~
+'hello'
+~~~
+{: .output}
+
+## Creating Variables
+
+This is great but not very interesting.
+To do anything useful with data, we need to assign its value to a _variable_.
+In Python, we use the equals sign `=` to assign a value (on the right) to a
+variable (on the left). The variable is created when a value is assigned to
+it.
+For example, we can track the weight of a patient who weighs 60.3 kilograms by
+assigning the value `60.3` to a variable `weight_kg`:
+
+~~~
+weight_kg = 60.3
+~~~
+{: .language-python}
+
+From now on, whenever we use `weight_kg`, Python will substitute the value we
+assigned to it. In layperson's terms, **a variable is a name for a value**.
+
+To identify and track a patient, we can assign them a unique identifier by
+storing it in a string:
+
+~~~
+patient_id = '001'
+~~~
+{: .language-python}
+
+In Python, variable names:
+
+ - can include letters, digits, and underscores;
+ - cannot start with a digit;
+ - are case-sensitive.
+
+This means that, for example:
+ - `weight0` is a valid variable name, whereas `0weight` is not;
+ - `weight` and `Weight` are different variables.
+
+## Using Variables in Python
+
+Once we have data stored in variables, we can make use of it in calculations.
+We may be interested in our patient's weight in pounds:
+
+~~~
+# There are 2.2 pounds per kilogram
+2.2 * weight_kg
+~~~
+{: .language-python}
+~~~
+132.66
+~~~
+{: .output}
+
+We might decide to add a prefix to our patient identifier:
+
+~~~
+patient_id = 'inflam_' + patient_id
+~~~
+{: .language-python}
+
+Variables must be created before they are used.
+
+## Using Built-in Functions
+
+To carry out common tasks with data and variables in Python,
+the language provides us with several built-in functions.
+To display information to the screen, we use the `print` function:
+
+~~~
+print(weight_lb)
+print(patient_id)
+~~~
+{: .language-python}
+~~~
+132.66
+inflam_001
+~~~
+{: .output}
+
+When we want to make use of a function, which is referred to as _calling_ a
+function, we follow its name by parentheses. The parentheses are important:
+If you leave them off, the function doesn't actually run.
+Sometimes we will include values or variables inside the parentheses.
+In the case of `print`, we include what we want to display.
+
+~~~
+print("hello")
+print("3")
+print(3)
+~~~
+{: .language-python}
+~~~
+hello
+3
+3
+~~~
+{: .output}
+
+We will learn more about how functions work and how to create our own in a
+later episode.
+
+We can display multiple things at once using only one `print` call:
+
+~~~
+print(patient_id, 'weight in kilograms:', weight_kg)
```

Comment:
I have thought about it, I hesitated, and I thought I shouldn't add any cognitive load... But, since you're suggesting, I'm happy to add it, specifying that we're seeing this just in passing (just so they've heard of it). Thanks for the suggestion!

Reply:
Done at 8d9dbaa043.